terraform/shared/shared.tf.json.template.py

```diff
@@ -524,6 +524,19 @@ def paren(s: str) -> str:
                     'value': 1,
                     'default_value': 0,
                 }
+            },
+            'clamscan': {
+                'name': config.qualified_resource_name('clamscan', suffix='.filter'),
+                # Patterns that include non-alphanumeric characters must be
+                # wrapped in double quotation marks ("")
+                'pattern': '"docker: clamscan"',
+                'log_group_name': '/aws/cwagent/azul-gitlab',
+                'metric_transformation': {
+                    'name': config.qualified_resource_name('clamscan'),
+                    'namespace': 'LogMetrics',
+                    'value': 1,
+                    'default_value': 0,
+                }
             }
```

Comment:
We may have lost the thread in the years since the ticket was filed:

Does this PR exhibit the [behavior](https://github.com/DataBiosphere/azul/issues/3895#issuecomment-1960588474) I clarified on the ticket just now?